Calibration/LumiAlCaRecoProducers/python/ALCARECOAlCaPCCRandom_cff.py

```diff
@@ -13,6 +13,10 @@
 siPixelDigisForLumiR = siPixelDigis.clone()
 siPixelDigisForLumiR.InputLabel = cms.InputTag("hltFEDSelectorLumiPixels")
 
+# Modify for if the phase 1 pixel detector is active
+from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
+phase1Pixel.toModify(siPixelDigisForLumiR, isUpgrade=cms.untracked.bool(True) )
```

Comment:
I may be missing something, but `siPixelDigisForLumiR` is cloned from `EventFilter.SiPixelRawToDigi.SiPixelRawToDigi_cfi.siPixelDigis`, which already has `isUpgrade` customized to `True` with `phase1Pixel` era
https://github.com/cms-sw/cmssw/blob/07706592fcafef0134ff379925cc496f4858209d/EventFilter/SiPixelRawToDigi/python/SiPixelRawToDigi_cfi.py#L21-L22

I'm just afraid that since presumably the `isUpgrade=False` at the moment, it will continue to be so after this PR, and the actual problem lies elsewhere (e.g. does the job configuration set the era to `Run2_2017`)?

(same for the other two config files)